src/app/pages/customerProfile/customerProfile.component.ts

```diff
@@ -9,23 +10,27 @@ import { Router } from '@angular/router';
 })
 
 export class CustomerProfileComponent implements OnInit {
+    urlId: string;
     isLoggedIn: boolean = false;
     loginPressed: boolean = false;
     editClicked: boolean = true;
-    customer: object;
+    ownProfile: any;
+    correctCustomer: boolean = false;
```

Comment:
You changed the one variable. CorrectCustomer makes no sense, ownProfile makes more sense. You should change back ownProfile to customer, and change correctCustomer to ownProfile.

Reply:
Also, stop using any.